ansible/roles_ocp_workloads/ocp4_workload_windows_node/tasks/workload.yml

```diff
@@ -0,0 +1,147 @@
+---
+# Implement your Workload deployment tasks here
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# Download SSH proxy container creator script
+- name: Download SSH proxy container creator script
+  get_url:
+    url: https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh
```

Comment:
Should this point to a specific tag to not potentially break in the future?

Reply:
Pointed to the most stable tag.